torch/fx/passes/_tensorify_python_scalars.py

```diff
@@ -244,13 +260,72 @@ def _sympy_interp(expr: sympy.Expr) -> MetaProxy:
                         )
 
                     node.replace_all_uses_with(replacement_proxy.node)
-
                     graph.erase_node(node)
+                    deleted.add(node)
 
     # DCE symbols (which are guaranteed to be pure) only
-    for proxy in reversed(expr_to_sym_proxy.values()):
-        if len(proxy.node.users) == 0 and proxy.node.op != "placeholder":
-            graph.erase_node(proxy.node)
+    dce(graph, deleted)
+
+    # Now do one more pass that specializes all symfloats we didn't manage
+    # to tensorify away.
+    nodes = list(graph.nodes)
+    for i, node in enumerate(nodes[:-1]):
+        with graph.inserting_before(
+            nodes[i + 1] if node not in placeholders else first_non_placeholder
+        ):
+            args = []
+            kwargs: Dict[str, Any] = {}
+            transform = False
+            for a in node.args:
+                if a in specialized_float_nodes:
+                    args.append(specialized_float_nodes[a])
+                elif (
+                    isinstance(a, fx.Node)
+                    and "val" in a.meta
+                    and isinstance(zf := a.meta["val"], torch.SymFloat)
+                    and isinstance(zf.node.expr, Symbol)
+                    and zf.node.hint is not None
+                ):
+                    transform = True
+                    args.append(float(zf))
+                else:
+                    args.append(a)
+
+            for k, v in node.kwargs.items():
+                if (
+                    isinstance(v, fx.Node)
+                    and "val" in v.meta
+                    and isinstance(zf := v.meta["val"], torch.SymFloat)
+                    and isinstance(zf.node.expr, Symbol)
+                    and zf.node.hint is not None
+                ):
+                    transform = True
+                    kwargs[k] = float(zf)
+                else:
+                    kwargs[k] = v
+
+            if (
+                callable(node.target)
+                and len(args) > 0
+                and all(isinstance(arg, (float)) for arg in args)
```

Comment:
nit: no parens around float

Reply:
Is there an easy way to add a rule so lintrunner will fix this for me automatically?